examples/react-signer/package.json

```diff
@@ -21,6 +21,7 @@
     "@dfinity/principal": "^2.1.3",
     "@dfinity/utils": "^2.5.1",
     "@radix-ui/themes": "^3.1.4",
+    "@slide-computer/signer": "^3.7.0",
```

Comment:
The line was removed in the previous commit, but it appears to have been reintroduced due to a merge conflict.
![image](https://github.com/user-attachments/assets/29a5da8d-32ca-42cf-b0bd-702bf0eb448e)

Reply:
Woops, removing it now.